cluster/definition.go

```diff
@@ -125,46 +121,42 @@ func (d Definition) NodeIdx(pID peer.ID) (NodeIdx, error) {
 	return NodeIdx{}, errors.New("unknown peer id")
 }
 
-// Sealed returns true if all operator signatures are populated and valid.
+// Sealed returns true if all config signatures are fully populated and valid. A "sealed" definition is ready for use in DKG.
 func (d Definition) Sealed() (bool, error) {
-	paramHash, err := d.HashTreeRoot()
+	configHash, err := d.ConfigHash()
 	if err != nil {
-		return false, errors.Wrap(err, "param hash")
+		return false, errors.Wrap(err, "config hash")
 	}
 
-	// Check that we a operator signature for each operator.
+	// Check that we have a valid operator signature for each operator.
 	for _, o := range d.Operators {
-		digest, err := digestEIP712(o.Address, paramHash[:], 0)
+		digest, err := digestEIP712(o.Address, configHash[:], 0)
 		if err != nil {
 			return false, err
 		}
 
-		var found bool
-		for _, sig := range d.OperatorSignatures {
-			if ok, err := verifySig(o.Address, digest[:], sig); err != nil {
-				return false, err
-			} else if ok {
-				found = true
-				break
-			}
-		}
-
-		if !found {
-			return false, nil
+		if ok, err := verifySig(o.Address, digest[:], o.ConfigSignature); err != nil {
```

Comment:
likewise verification of ENR signatures can also be added

Reply:
yeah